tests/test_mock_controller.py

```diff
@@ -531,8 +639,14 @@ async def test_setTemperature(self):
         await self.write(f"setTemperature:\n temperature: {temperature}\n")
         self.data = await self.read()
         self.assertEqual(self.data["OK"]["Timeout"], self.mock_ctrl.long_timeout)
+
+        # Set the TAI time in the mock controller for easier control
+        self.mock_ctrl.current_tai = salobj.current_tai()
+        # Set the mock device statuses TAI time to the mock controller time for easier control
+        self.mock_ctrl.thcs.command_time_tai = self.mock_ctrl.current_tai
         # Give some time to the mock device to set the temperature.
-        await asyncio.sleep(0.2)
+        self.mock_ctrl.current_tai = self.mock_ctrl.current_tai + 0.2
+
```

Comment:
I hope you are testing the positions of your mock actuators somewhere. (One of the big advantages of the new way of managing time is you can easily specify a given time and know what position to expect). I'm not seeing it in the diffs above.

Reply:
You don't see that in the differences because this is part of the test_mock_controller test and it was already added there when the timed loop was added. There are no separate test files for the actuators yet and I will consider adding them.